### PR TITLE
Feature/certs

### PR DESCRIPTION
**Pull request by steeling**
Separate out the CA cert for easier rotation and adding partner operators

### CODE REVIEW

File: config/apply-certs.sh [hunk 1]
```diff
@@ -8,17 +8,29 @@ DIR="$(pwd)"
...
 JWT_PUBLIC_CERTS_DIR="$DIR/jwt-public-certs"
+UPLOAD_CA_KEY=false
 
 # Delete previous secrets in case they have changed.
 kubectl create namespace "$NAMESPACE" || true
 
 kubectl delete secret cockroachdb.client.root || true
 kubectl delete secret cockroachdb.client.root --namespace "$NAMESPACE" || true
 kubectl delete secret cockroachdb.node --namespace "$NAMESPACE" || true
+kubectl delete secret cockroachdb.node --namespace "$NAMESPACE" || true
```
Comment (BenjaminPelletier) [inline config/apply-certs.sh]: Looks like this line is a duplicate of the previous one?

File: config/build.sh [hunk 2]
```diff
@@ -15,3 +15,4 @@ cd "${BASEDIR}"
 docker build . -t $DOCKER_URL/dss:$VERSION
 
 docker push $DOCKER_URL/dss:$VERSION
+echo $DOCKER_URL/dss:$VERSION
```
Comment (BenjaminPelletier) [inline config/build.sh]: nit: Maybe add a prefix mentioning what's being done (e.g., `echo Pushing docker image $DOCKER_URL/dss:$VERSION`)

File: config/make-certs.py [hunk 3]
```diff
@@ -128,6 +133,9 @@ def main():
         '--ca-key', cr.ca_key_file]
         + node_addresses)
 
+    subprocess.check_call(['rm', os.path.join(cr.node_certs_dir, 'ca.crt')])
```
Comment (BenjaminPelletier) [inline config/make-certs.py]: Maybe os.remove instead of shelling out?

File: config/templates/cluster-init-secure.yaml [hunk 4]
```diff
@@ -16,7 +16,14 @@ spec:
         imagePullPolicy: IfNotPresent
         volumeMounts:
         - name: client-certs
```
Comment (BenjaminPelletier) [inline config/templates/cluster-init-secure.yaml]: Just to verify, should this name be duplicated for both items?  (same question elsewhere as well)

**Comment by jgrossmac** [comment]
Looks good to me other than a few duplicates mentioned by @BenjaminPelletier

File: config/templates/cluster-init-secure.yaml (hunk 4 above)
Comment (steeling) [reply config/templates/cluster-init-secure.yaml]: ya to mount multiple volumes to the same directory you have to call out each file specifically

File: config/make-certs.py (hunk 3 above)
Comment (steeling) [reply config/make-certs.py]: done

File: config/build.sh (hunk 2 above)
Comment (steeling) [reply config/build.sh]: done

File: config/apply-certs.sh (hunk 1 above)
Comment (steeling) [reply config/apply-certs.sh]: yup, done

**Pull request merged**